src/benchmarks/micro/README.md

```diff
@@ -1,287 +1,90 @@
-# Benchmarks
+# Micro Benchmarks
 
-This repo contains various .NET benchmarks. It uses BenchmarkDotNet as the benchmarking engine to run benchmarks for .NET, .NET Core, CoreRT and Mono. Including private runtime builds.
+This folder contains micro benchmarks that test the performance of .NET Runtime(s).
 
-## BenchmarkDotNet
+## Tooling
 
-Benchmarking is really hard (especially microbenchmarking), you can easily make a mistake during performance measurements.
-BenchmarkDotNet will protect you from the common pitfalls (even for experienced developers) because it does all the dirty work for you:
+To run the benchmarks, you need to [download dotnet cli](../../../docs/prerequisites.md) first.
```

Comment:
or use the python script? the other doc implies you don't need to get CLI yourself.